README.md

```diff
@@ -34,6 +34,24 @@ The barcode column can take values following the format barcode[0-9][0-9] or NB[
 
 Note: Basecalling and demultiplexing may also be performed if not already done on GridION/MinIT.
 
+## Basic usage using Dockerfile/image
+
+The Dockerfile/image is made to be used with Docker-out-of-Docker (DooD), where we mount `/var/run/docker.sock` into our container and our container will start new containers that are "siblings" to the one we started it from, and thus runs them in the context of the host. Due to this we have to use absolute paths when mounting volumes and setting `--input_dir` and `--sample_names`. 
```

Comment:
I find it confusing to refer to it as Dockerfile/image, I would just write docker image because this is used for DooD.

Reply:
Thank you!